Add `offline-url` documentation (#7193)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/vuln-detector.rst

   :depth: 1
   :backlinks: none

+---------------------------+--------------------------------------+
| Options                   | Allowed values                       |
+===========================+======================================+
| `enabled`_                | ``yes``, ``no``                      |
+---------------------------+--------------------------------------+
| `feed-update-interval`_   | Positive number + Time unit suffix   |
+---------------------------+--------------------------------------+
| `index-status`_           | ``yes``, ``no``                      |
+---------------------------+--------------------------------------+

> | `offline-url`_            | ``<path>`` (starting with \file://)  |

Reviewer: We could use `file://<LOCAL_FILE_PATH>` and `http[s]://<URL>` instead

Author: Good catch!

+---------------------------+------------------------------------+
| `index-status`_           | ``yes``, ``no``                    |
+---------------------------+------------------------------------+
| `offline-url`_            | ``<path>`` (starting with file://) |
|                           | or ``<URL>``                       |

> |                           | (starting with http:// or https://)|

Reviewer 2: I would remove the hyperlinks here. 

Here's an example I found within the documentation 

https://documentation.wazuh.com/current/user-manual/user-administration/single-sign-on/administrator/pingone.html
![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/7653ea86-4603-4625-a3f8-61381b33fc32)

Author: Done. Thx!

| **Allowed values** | ``yes``, ``no``             |
+--------------------+-----------------------------+

offline-url

> ^^^^^^^^^^^

Author: Done!

> Allows to specify a path or URL for offline content access.

Reviewer: ```suggestion
File path or URL for offline content access.
```

Author: Done!

+--------------------+-----------------------------+
| **Default**        | Empty                       |
+--------------------+-----------------------------+

> | **Allowed values** | ``<path>``, ``<URL>``       |

Reviewer: We could use `file://<LOCAL_FILE_PATH>` and `http[s]://<URL>` instead

Author: I agree!

+--------------------+-----------------------------+

``<path>``: A local file path starting with file://, pointing to offline content stored on the device.

> ``<URL>``: A URL starting with either http:// or https://, pointing to online content accessible via the internet or local network.

Reviewer 2: I would remove the hyperlinks here.

Author: Done. Thx!

> ``<URL>``: A URL starting with either \http:// or \https://, pointing to online content accessible via the internet or local network.

Reviewer: It couldn't be "via the internet" if we're referring to offline content.

> ``<URL>``: A URL starting with either \http:// or \https://, pointing to online content accessible via the internet or local network.

Reviewer: ```suggestion
``<URL>``: A URL starting with either \http:// or \https://, pointing to local network content or online content accessible via the internet.
```

Author: Done!
